### PR TITLE
CASMINST-5197

### PR DESCRIPTION
**Pull request by dzou-hpe**
# Description

- force k8s sync after uploading workflow files

# Test
fanta

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: upgrade/scripts/upgrade/ncn-upgrade-worker-storage-nodes.sh [hunk 1]
```diff
@@ -101,6 +101,7 @@ fi
 
 function uploadWorkflowTemplates() {
     "${basedir}"/../../../workflows/scripts/upload-worker-rebuild-templates.sh
+
```
Comment (mharding-hpe) [inline upgrade/scripts/upgrade/ncn-upgrade-worker-storage-nodes.sh:104]: No other functions end with a blank line that I can see.

```suggestion
```

**Review by mharding-hpe — APPROVED** [review]

**Comment by mharding-hpe** [comment]
Is this also needed in release/1.3 ?

**Review by rustydb — APPROVED** [review]

**Comment by rustydb** [comment]
> Is this also needed in release/1.3 ?

I'd assume yes.
@dzou-hpe ?

**Pull request merged**